ouroboros-consensus/ouroboros-consensus-mock/test/Test/ThreadNet/Praos.hs

```diff
@@ -122,5 +122,6 @@ prop_simple_praos_convergence TestSetup
                                       (HardFork.defaultEraParams
                                         k
                                         slotLength)
+                                      0
```

Comment:
Hmmm, I must have lost part of the diff. Will fix this degenerate argument shortly.